src/ui/pages/History/AccelerateTransactionDialog/CancelTx/CancelTx.tsx

```diff
@@ -56,6 +57,7 @@ function CancelTxContent({
   onDismiss: () => void;
   onSuccess: () => void;
 }) {
+  const { preferences } = usePreferences();
```

Comment:
Unfortunately, useQuery doesn't work inside <Content/> right now, so I placed this preferences check outside of SignButtons. Will be glad to discuss better way if needed.